site/content/en/latest/tasks/operations/standalone-deployment-mode.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "Standalone Deployment Mode"
+---
+
+{{% alert title="Notice" color="warning" %}}
+
+Standalone mode is an experimental feature, please **DO NOT** use it in production.
+
+{{% /alert %}}
+
+Envoy Gateway also supports running in standalone mode. In this mode, you can configure the
+Envoy Gateway to use different file provider and infrastructure provider combinations.
```

Comment:
Not a strong opinion - I suggest we remove this and rephrase the standalone mode as the only support combination is file provider and host infrastructure. It would be confusing for readers.